src/DevChatter.Bot.Core/Data/Model/QuizQuestion.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace DevChatter.Bot.Core.Data.Model
+{
+    public class QuizQuestion : DataEntity
+    {
+        public string MainQuestion { get; set; }
+        public string Hint1 { get; set; }
+        public string Hint2 { get; set; }
+        public string CorrectAnswer { get; set; }
+        public string WrongAnswer1 { get; set; }
+        public string WrongAnswer2 { get; set; }
+        public string WrongAnswer3 { get; set; }
```

Comment:
Is it a good idea to hard-code the number of wrong answers and hints? It would be more flexible to use a collection of some sort

Reply:
For now, we're going to be 4, and we may update the model eventually. Locking on 4 will make some things easier when we give this a UI.